alchemy/src/cloudflare/oauth.ts

```diff
@@ -70,7 +70,7 @@ export const getRefreshedWranglerConfig = singleFlight(() =>
 );
 
 const getWranglerConfigPath = memoize(async () => {
-  const xdgConfigDir = xdgAppPaths(".wrangler").config();
+  const xdgConfigDir = envPaths("wrangler", { suffix: "" }).config;
```

Comment:
The `suffix` option is important here because otherwise `envPaths` will append a `-nodejs` suffix onto the config path. The original intent of this package is for storing new things, not necessarily looking up existing. Still, we can get the same behavior by passing in an empty string.

Reply:
```suggestion
  const xdgConfigDir = envPaths(".wrangler", { suffix: "" }).config;
```